### DISCUSSION TITLE
How to delete added services in page?

### DISCUSSION THREAD

**Discussion by LuanShiZhiJian** [apache/skywalking#11617] [Q&A]
these service are no longer used, so how to delete it? please advice, thanks.

![image](https://github.com/apache/skywalking/assets/41558924/c8e46772-7b53-4d3b-9c49-ef2f9702a5af)

**Comment by wu-sheng** [comment]
TTL would take care of that automatically. https://skywalking.apache.org/docs/main/next/en/setup/backend/ttl/#ttl

Observability system is not a CRUD management platform.
